__tests__/spec/sanity-checks.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-env jest */
+var request = require('supertest')
+var app = require('../../app.js')
+var path = require('path')
+var fs = require('fs')
+var assert = require('assert')
+
+/**
+ * Basic sanity checks on the dev server
+ */
+describe('The Prototype Kit', () => {
+  test('should generate assets into the /public folder', () => {
```

Comment:
I believe we're consistently using `it()` over `test()` in our other products – unless there's a good reason not to, it'd be good to follow suit.